src/CMakeLists.txt

```diff
@@ -362,6 +362,8 @@ if(MINGW)
   set(CPACK_NSIS_MUI_UNWELCOMEFINISHPAGE_BITMAP "${CPACK_NSIS_MUI_WELCOMEFINISHPAGE_BITMAP}")
   set(CPACK_NSIS_CREATE_ICONS_EXTRA "CreateShortCut '$SMPROGRAMS\\\\$STARTMENU_FOLDER\\\\${PROGNAME}.lnk' '$INSTDIR\\\\${PROGNAME}.exe'")
   set(CPACK_NSIS_DELETE_ICONS_EXTRA "Delete '$SMPROGRAMS\\\\$START_MENU\\\\${PROGNAME}.lnk'")
+  set(CPACK_NSIS_EXTRA_INSTALL_COMMANDS "ExecWait 'Taskkill /IM keepassxc-proxy.exe /F'")
```

Comment:
This needs to be ```CPACK_NSIS_EXTRA_PREINSTALL_COMMANDS``` so they execute before any files are written and need to add an extra command to kill keepassxc.exe itself. Here is the desired entry:

```
set(CPACK_NSIS_EXTRA_PREINSTALL_COMMANDS
          "ExecWait 'Taskkill /IM keepassxc-proxy.exe /F'
          ExecWait 'Taskkill /IM keepassxc.exe /F'")
```